### PR TITLE
Added states for Deebot N79; including auto, spot, and edge.

### PR DESCRIPTION
**Pull request by clayauld**
I added extra support for my Deebot N79 with cleaning states "auto", "spot", and "edge". These are subsets of the 'cleaning' state and the card didn't adjust to these properly. This PR fixes that.

### CODE REVIEW

File: src/styles.js [hunk 1]
```diff
@@ -123,7 +127,8 @@ export default css`
     opacity: 100%;
   }
 
-  .vacuum.docked {
+  .vacuum.docked,
+  .vacuum.idle {
```
Comment (denysdovhan) [inline src/styles.js:131]: This was intentional. Idle without opacity has to indicate that vacuum is not in the proper place.

**Review by denysdovhan — CHANGES_REQUESTED** [review]

File: src/styles.js (hunk 1 above)
Comment (clayauld) [reply src/styles.js:131]: Gotcha. I will fix this.

Comment (clayauld) [reply src/styles.js]: Current push should resolve this.

**Review by clayauld — COMMENTED** [review]
Looks good

**Review by denysdovhan — APPROVED** [review]

**Pull request merged**